src/main/java/com/matt/forgehax/mods/MatrixNotifications.java

```diff
@@ -194,15 +236,11 @@ private static void postAsync(final String url, final JsonElement json) {
     EXECUTOR.submit(() -> {
       try {
         HttpResponse res = post(url, json);
-        if (res.getStatusLine().getStatusCode() != 200) {
+        if (res.getStatusLine().getStatusCode() != 200 && res.getStatusLine().getStatusCode() != 204) {
```

Comment:
This will always throw an error. Change `&&` to `||`

Reply:
Why would it always throw? I know it matters little but works fine for me, using both Discord and Matrix as a provider.

Only throw if different from both 200 and 204. Using an `||` would make it always fire, even in case of Response Code 200 (success) : first check is false (not different from 200), but 2nd turns out true and an exception is fired. In case of any other response code, only the first is checked, returns true (different from 200) and exception is fired without checking the 2nd one.

This is needed for direct Discord webhooks, because they reply a lot with 204, triggering many "errors". Could be redone with its own `if` statement I think

Reply:
oops you're right. brain fart moment